ui/shared/utils/constants.js

```diff
@@ -590,6 +591,66 @@ const ORDERED_VEP_CONSEQUENCES = [
     value: 'INV',
     group: VEP_GROUP_SV,
   },
+  {
+    description: 'A loss of function effect',
+    text: 'Loss of function',
+    value: 'LOF',
+    group: VEP_GROUP_SV_CONSEQUENCES,
+  },
+  {
+    description: 'A loss of function effect via intragenic exonic duplication',
+    text: 'Loss Of Functionn Duplication',
+    value: 'DUP_LOF',
+    group: VEP_GROUP_SV_CONSEQUENCES,
+  },
+  {
+    description: 'A copy-gain effect',
+    text: 'Copy Gain',
+    value: 'COPY_GAIN',
+    group: VEP_GROUP_SV_CONSEQUENCES,
+  },
+  {
+    description: 'Having genes partially overlapped by the SV\'s duplication',
+    text: 'Partial Duplication',
+    value: 'DUP_PARTIAL',
+    group: VEP_GROUP_SV_CONSEQUENCES,
+  },
+  {
+    description: 'A multiallelic SV predicted to have a LOF, DUP_LOF, COPY_GAIN, or DUP_PARTIAL',
```

Comment:
use the human-readable names not the coded names, i.e `Loss of function` instead of `LOF` and add the word "effect" at the end